src/libclient/measurement/ping/ping_linux.cpp

```diff
@@ -291,12 +309,12 @@ Result Ping::result() const
     res.insert("round_trip_max", max);
     res.insert("round_trip_stdev", stdev);
     res.insert("round_trip_ms", roundTripMs);
-    res.insert("round_trip_sent", m_pingsSent);  // count successfull pings only
-    res.insert("round_trip_received", m_pingsReceived);
+    res.insert("round_trip_sent", tmpPingsSent);  // count successfull pings only
+    res.insert("round_trip_received", tmpPingsReceived);
 
     if (m_pingsSent > 0)
     {
-        res.insert("round_trip_loss", (m_pingsSent - m_pingsReceived) / m_pingsSent);
+        res.insert("round_trip_loss", (tmpPingsSent - tmpPingsReceived) / tmpPingsSent);
```

Comment:
This is still an integer division (the result will be 0 or 1, not the exact percentage).